db/migrate/20160330000130_add_foreign_key_constraints.rb

```diff
@@ -0,0 +1,33 @@
+class AddForeignKeyConstraints < ActiveRecord::Migration
+  def change
+    add_foreign_key :absences, :student_school_years
```

Comment:
:+1: yeah I think it has to be a manual walk through the semantics of each relationship, I don't think anyone knows all the relationships off the top of their head (@alexsoble might still!).  Not sure I have any better recommendation but am happy to be another set of eyes, so if that's helpful ever feel free to ping me!